README.md

```diff
@@ -30,7 +30,8 @@ Please note: files under `app/etc/` are processed in alphabetical order.
 
 ##Exclude route from parameter removal process
 
-In a new config file found in `app/etc/`. You may add an optional `<exclude_list>` that can be used to exempt or exclude a route/URI from the PageCacheParam process. Any URI that contains the string in the `route` attribute will be excluded from the PageCacheParams processor.
+In a new config file found in `app/etc/`. You may add an optional `<exclude_list>` that can be used to exempt or exclude a path/URI from the PageCacheParam process. Any URI that contains the string in the `path` attribute will be excluded from the PageCacheParams processor.
+Additionally default exclusions can be overwritten using the `ignore` attribute.
```

Comment:
nit: Well, they're also processed in alphabetical order, so not just default.  But this is fine.